spec/classes/puppet_server_config_spec.rb

```diff
@@ -714,6 +714,53 @@
         end
       end
 
+      describe 'alow jetty specific server threads' do
```

Comment:
Could you move these tests to `spec/classes/puppet_server_puppetserver_spec.rb` where we have all the tests just for puppetserver?